src/comps/pages/catalog/OrgCard.tsx

```diff
@@ -1,99 +1,250 @@
+                        },
+                        height: "450px",
+                        display: "flex",
+                        flexDirection: "column",
+                        overflow: "visible",
+                        padding: "30px",
+                        background: "rgba(31, 31, 31, 0.5)",
+                        justifyContent: "flex-start",
+                        zIndex: 1,
+                    }}
+                    onClick={() => {
+                        navigate(`/${organization.url}`);
+                    }}
+                >
+                    <div
+                        style={{
+                            background:
+                                "linear-gradient(90deg, rgba(0, 0, 0, 0) 0%, rgba(255, 255, 255, 0) 0%, rgba(143, 143, 143, 0.67) 50%, rgba(0, 0, 0, 0) 100%)",
+                            width: "150px",
+                            height: "1px",
+                            position: "absolute",
+                            bottom: "0px",
+                            right: "65px",
+                            opacity: 0.25,
+                        }}
+                    ></div>
+
+                    <div
+                        style={{
+                            background:
+                                "linear-gradient(90deg, rgba(0, 0, 0, 0) 0%, rgba(255, 255, 255, 0) 0%, rgba(143, 143, 143, 0.67) 50%, rgba(0, 0, 0, 0) 100%)",
+                            width: "300px",
+                            height: "1px",
+                            position: "absolute",
+                            top: "0px",
+                            opacity: 0.5,
+                        }}
+                    ></div>
+
+                    <Box
+                        sx={{
+                            width: "100%",
+                            display: "flex",
+                            flexDirection: "column",
+                        }}
+                    >
+                        <Box
+                            sx={{
+                                position: "relative",
+                                top: -50,
+                                marginBottom: "-20px",
+                            }}
+                        >
+                            <Box
+                                sx={{
+                                    width: "130px",
+                                    height: "130px",
+                                    borderRadius: "20px",
+                                    position: "absolute",
+                                    boxShadow:
+                                        "inset 0 0 1px 1px rgba(255, 255, 255, 0.15)",
+                                    zIndex: 10,
+                                }}
+                            ></Box>
+
+                            <Avatar
+                                src={organization.picture || ""}
```

Comment:
Remind me, does this work for clubs with no pictures? Example: https://epsilon.stuysu.org/httpsepsilonstuysuorgjuniorsing (one day we'll filter out these icky urls)

Reply:
yup